src/UI/Art/SpriteArtist/InfoView.java

```diff
@@ -139,26 +149,24 @@ private BufferedImage renderPage(ArrayList<InfoPage> pages, int drawingWidth)
       {
         case CO_HEADERS: // Draw all CO image headers and brief status text for each CO
         {
-          Commander[] coList = myControl.getGame().commanders;
+          Army[] armyList = myControl.getGame().armies;
 
-          for( Commander thisCO : coList )
-          {
-            BufferedImage statusHeader =
-                SpriteLibrary.createTransparentSprite(drawingWidth, CommanderOverlayArtist.OVERLAY_HEIGHT + 5);
-            Graphics headerG = statusHeader.getGraphics();
-            CommanderOverlayArtist.drawCommanderOverlay(headerG, thisCO, 0, true);
+          int ySpaceNeeded = 0;
+          final int overlayHeight = SpriteLibrary.getCoOverlay(armyList[0].cos[0], true).getHeight();
+          for( Army thisArmy : armyList )
+            // Draw a little generously
+            ySpaceNeeded += (thisArmy.cos.length + 1) * overlayHeight;
```

Comment:
Generously? Can we not draw with exactness?

Reply:
As I recall, I was trying to account for the height of the brief summary, which varies by the input font.
Having a little extra space is unlikely to hurt anything.

Reply:
Seems like it wouldn't be too hard to account for that space. As is, it looks like you are adding an extra overlay's worth of height for each team in play. Maybe a bit excessive.

Reply:
This has been addressed on `presentationFixes`, I think?
![image](https://user-images.githubusercontent.com/8674241/168440774-bd514328-6ac9-40f0-8ca3-2d889c986c1b.png)